css-layout-classes/styles.css

```diff
@@ -0,0 +1,32 @@
+body {
+  font-family: 'Arial', sans-serif;
+}
+
+.column-one-fourth {
+  max-width: 25%;
+}
+
+.column-one-fourth > img {
+  max-width: 200px;
+}
```

Comment:
Instead of individually sizing the images, I would just set a `width` of `100%` on them, as the directions mention:
![image](https://user-images.githubusercontent.com/32880187/145121642-1a733c75-d28d-4f06-aea3-f9fd95383f72.png)